contracts/CO30_Thomas_Intervention.VTF_Lybor/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,14 @@
+Bad actors in the area have banded together to try and hold industrial areas in the AO hostage so that they can create an area of their own control. Outside forces have been unable/unwilling to enter the area due to the hilly terrain, that is until tomorrow at 1500 when a outside equipped paramilitary force will begin moving into the area. It is believed that the paramilitary force won't spare much thought to hostages as their primary objective will be the elimination of the insurgents, and recapture of the industrial areas.
```

Comment:
```suggestion
Bad actors in the area have banded together to try and hold industrial areas in the AO hostage so that they can create an area of their own control. Outside forces have been unable/unwilling to enter the area due to the hilly terrain, that is until tomorrow at 1500 when an outside equipped paramilitary force will begin moving into the area. It is believed that the paramilitary force won't spare much thought to hostages as their primary objective will be the elimination of the insurgents, and recapture of the industrial areas.
```